app/forms/openid_connect_authorize_form.rb

```diff
@@ -161,8 +161,15 @@ def error_redirect_uri
   end
 
   def scopes
-    return OpenidConnectAttributeScoper::VALID_SCOPES if ialmax_requested? || ial2_requested?
-    OpenidConnectAttributeScoper::VALID_IAL1_SCOPES
+    values = if ialmax_requested? || ial2_requested?
+      OpenidConnectAttributeScoper::VALID_SCOPES
+    else
+      OpenidConnectAttributeScoper::VALID_IAL1_SCOPES
+    end
+
+    values += %w[profile:verified_at] if service_provider.ial >= 2
```

Comment:
`profile:verified_at` is a weird scope.... I could just add it to `VALID_IAL1_SCOPES` and be done with it? But we only want to expose it to SPs that are IAL2+.... I don't like how messy this is, very open to feedback

Reply:
Yeah, that is tricky. If we added it to `VALID_IAL_1_SCOPES` we could back it up with a validator that added an error if the service provider is not IAL2 and that scope is present.

Reply:
Ok cool, I'll go with that approach and add an error